### ISSUE TITLE
Bug:  Duplicated step checking is missing

### ISSUE THREAD

**Issue by jzaratei** [vitalets/playwright-bdd#74]
**Given**
```
export default defineConfig({
  ...
  project : {
      name: "login-chrome",
      testDir: defineBddConfig({
        outputDir: ".features-gen/login",
        importTestFrom: "page-objects/basePage.ts",
        paths: ["./features/login.feature"],
        require: ["page-objects/*.ts"],
        featuresRoot: "./features",
        verbose: true,
      }),
    },
    {
      name: "Regresion - Chrome",
      testDir: defineBddConfig({
        outputDir: ".features-gen/regression",
        importTestFrom: "page-objects/basePage.ts",
        paths: ["./features/todo.feature"],
        require: ["page-objects/*.ts"],
        featuresRoot: "./features",
        verbose: true,
      }),
      dependencies: ["login-chrome"],
   }
```
```

`
  @Then('page screenshot matches previous one')
  async matchScreenshot() {
    await expect(this.page).toHaveScreenshot();
  }

  @Then('page screenshot matches previous one')
  async matchScreenshotv2() {
    await expect(this.page).toHaveScreenshot();
  }`
```

**When**
I execute npx bddgen

**Then**
no error msg is displayed and the test cases are not generated

**Expected behavior**
Error: Several step definitions found for text: page screenshot matches previous one (features\todopage.feature)

**Isolated demo**
(https://github.com/vitalets/playwright-bdd-example/pull/7)

**Environment**

```
```

**Comment by vitalets** [comment]
@jzaratei thanks for reporting! Will fix in the nearest release.

**Comment by vitalets** [comment]
Released in v5.6.0.
  Please re-check on your side and report if there are any issues.

**Comment by jzaratei** [comment]
> Released in v5.6.0. Please re-check on your side and report if there are any issues.

All is working fine!
